crates/stc_ts_file_analyzer/tests/visualize/types/typeLit/1.swc-stderr

```diff
@@ -30,4 +30,4 @@ Error:
    `----
 
 Error: 
-  > (string | null)
+  > string
```

Comment:
I think this is wrong

Reply:
because option is not apply
This is why #436 should be passed.